src/Loop.php

```diff
@@ -178,11 +178,18 @@ public static function repeat($interval, callable $callback, $data = null)
     /**
      * Execute a callback when a stream resource becomes readable or is closed for reading.
      *
+     * Warning: Closing resources locally, e.g. with `fclose`, might not invoke the callback. Be sure to `cancel` the
+     * watcher when closing the resource locally. Drivers might choose to notify the user in a debug mode if there are
```

Comment:
The emphasis is on removing "in a debug mode". Just keep it out, it has no business here.